Adds Tenancy/Contacts and Contact Groups

fr3nch13 opened the pull request:
You're right, it is pretty trivial to create new Entity definitions. 

Anyway, I needed contacts and contact groups, so I figured I'd add them here. As I work through my project, and find other entities that have a relationship with contacts, I'll open another PR to add those relationships.

thde commented:
@fr3nch13 thank you! Could you rebase your branch? I'll create a release as soon as this one is merged.

fr3nch13 commented:
Done. I'll most likely be adding more entities/relationships in the coming days, but a release with this one would be greatly appreciated!

thde commented:
It still creates conflicts. So I wait for now.

fr3nch13 commented:
In Github it's showing:

No conflicts with base branch
Changes can be cleanly merged.

Am I missing somethings?

fr3nch13 commented:
From my end, the merge conflict was in the `spec/netbox_client_ruby/api/dcim/site_spec.rb` because I updated it to use the `described_class`. But that was the only conflict.

When I rebased, everything was resolved on my end, so I'm not able to see what you're seeing.

fr3nch13 commented:
Can we approve this PR? Is there anything else you need from me?

thde commented:
I already did that two weeks ago, but the branch still had conflicts:
> This branch cannot be rebased due to conflicts

I rebased it and merged your changes in https://github.com/ninech/netbox-client-ruby/pull/96

fr3nch13 commented:
Thank you @thde i really appreciate it. I'm sure I'll have more changes and more PRs. :-)
